TrippyGL.Fonts.Building/FontBuilderExtensions.cs

```diff
@@ -76,7 +56,7 @@ public static TrippyFontFile CreateFontFile(ReadOnlySpan<string> fontFiles, floa
         {
             IGlyphSource[] glyphSources = new IGlyphSource[fontFiles.Length];
             for (int i = 0; i < glyphSources.Length; i++)
-                glyphSources[i] = new FontGlyphSource(FontInstance.LoadFont(fontFiles[i]), size, firstChar, lastChar);
+                glyphSources[i] = new FontGlyphSource(new FontCollection().Add(fontFiles[i]).CreateFont(size), firstChar, lastChar);
```

Comment:
Future note: Change depending on how [the other conversation](https://github.com/SilkCommunity/TrippyGL/pull/5/files#r1122172273) resolves.